src/components/o-modal/index.js

```diff
@@ -0,0 +1,83 @@
+import PropTypes from '../../js/prop-types';
+import BaseComponentGlobal from '../../js/abstract/base-component-global';
+import defineOnce from '../../js/define-once';
+import styles from './index.scss';
+import template from './_template';
+
+class AXAModal extends BaseComponentGlobal {
+  static tagName = 'axa-modal';
+  static propTypes = {
+    open: PropTypes.bool,
+  };
+
+  set open(value) {
+    this.setAttribute('open', value);
+  }
+
+  get open() {
+    return this.getAttribute('open');
+  }
+
+  closeModal() {
+    this.removeAttribute('open');
+  }
+
+  closeModalByClickingOutside(event) {
+    if (event.target === this.querySelector('.js-modal')) this.closeModal();
+  }
+
+  closeModalOnEsc(event) {
+    const key = event.key || event.keyCode;
+    if (key === 'Escape' || key === 'Esc' || key === 27) {
+      this.closeModal();
+    }
+  }
+
+  closeModalFunction = () => this.closeModal();
```

Comment:
did u test this on IE11?

Reply:
due to the proptype chain issue the new polyfill have?

Reply:
I requested OpenPc to test exactly that. But that will probably take some time.

Reply:
Testing on IE should no longer be necessary, since the implementation of this method changed (naturally, not as workaround).